### ISSUE TITLE
Question: Is there a way to mark subdirectories as not part of the subrepo?

### ISSUE THREAD

**Issue by aydun** [ingydotnet/git-subrepo#275]
I'm working with a structure like this:
```
mainrepo/
   dir1/
   dir2/
   subrepo/
      dir3/
      dir4/   <- Not part of subrepo
```
It's a third-party structure so I can't change the layout.  I want `mainrepo/subrepo/dir4` to be treated as part of `mainrepo`, not `subrepo`.  Is that possible?

**Comment by grimmySwe** [comment]
@aydun I don't think that is possible. Is there a reason that you need it located at that level?

**Comment by aydun** [comment]
@grimmySwe yes ... the repo holds deployed code for a drupal website.  In the previous tree, `subrepo` is the root of a complex module where most of the code is in one repo that I fork, but some directories (`subrepo/dir4`) are not part of that repo so should not be part of the branch created by a `git subrepo branch`.

I think I can work around that by making `subrepo/dir4` a subrepo itself.

However, trying that has hit an error when doing a `git subrepo branch`. 

What is the recommended version of subrepo?

**Comment by grimmySwe** [comment]
@aydun We are working on releasing version 0.4.0, so for versioning I would recommend that you use issue/216-G, that should contain almost everything that is prepared for the upcoming release.
